Investigation SIGKILL

```
2025-12-12 01:11:24.469622013 +0100 CET [web-1] [2025-12-12 00:11:24 +0000] [30] [CRITICAL] WORKER TIMEOUT (pid:31)
2025-12-12 01:11:25.639139169 +0100 CET [web-1] [2025-12-12 00:11:25 +0000] [575] [INFO] Booting worker with pid: 575
2025-12-12 01:11:25.516087546 +0100 CET [web-1] [2025-12-12 00:11:25 +0000] [30] [ERROR] Worker (pid:31) was sent SIGKILL! Perhaps out of memory?
2025-12-12 01:11:28.181695228 +0100 CET [web-1] INFO:app:Rechargement des jobs du scheduler...
2025-12-12 01:11:28.178820247 +0100 CET [web-1] INFO:apscheduler.scheduler:Scheduler started
````

--- a/app.py
+++ b/app.py
@@ -19,7 +19,7 @@ from sqlalchemy import (create_engine)
 from sqlalchemy.orm import declarative_base, sessionmaker
 from apscheduler.schedulers.background import BackgroundScheduler
 from apscheduler.triggers.cron import CronTrigger
-from apscheduler.executors.pool import ProcessPoolExecutor
+from apscheduler.executors.pool import ThreadPoolExecutor
 from zoneinfo import ZoneInfo
 from database.database_manager import DatabaseManager
 from database.models import OtpConfiguration, UserSchedule, SyncLog
@@ -30,7 +30,7 @@ Base = declarative_base()
 
 # Instance globale du scheduler APScheduler
 scheduler = BackgroundScheduler(executors={
-    'default': ProcessPoolExecutor(max_workers=5)
+    'default': ThreadPoolExecutor(max_workers=2)
 })
 
 # Déterminer le répertoire du script

